modules/deploying-models-using-multiple-gpu-nodes.adoc

```diff
@@ -0,0 +1,266 @@
+endif::[]
+ifdef::upstream[]
+* You have enabled the operators for your GPU type, such as Node Feature Discovery Operator, NVIDIA GPU Operator, and so on. For more information about enabling accelerators, see link:{odhdocshome}/working-with-accelerators[Working with accelerators^].
+endif::[]
+
+** You are using one of the following GPU types: `nvidia.com/gpu` (default), `intel.com/gpu`, `amd.com/gpu`, `habana.ai/gaudi`, or you have specified another GPU type in the annotations of the `InferenceService`. 
+** Specify the GPU type through either the `ServingRuntime` or `InferenceService`. If the GPU type specified in the `ServingRuntime` differs from what is set in the `InferenceService`, both GPU types are assigned to the resource and can cause errors. 
+
+ifndef::upstream[]
+* You have set up model-serving in raw deployment mode. For more information about raw deployment, see link:{rhoaidocshome}{default-format-url}serving_models/serving-large-models_serving-large-models#deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode_serving-large-models[Deploying models on single node {openshift-platform} using KServe raw deployment mode]
+endif::[]
+ifdef::upstream[]
+* You have set up model-serving in raw deployment mode.
+endif::[]
+
+* You have only 1 head pod in your setup. The replica count for the head pod can be adjusted using the `min_replicas` or `max_replicas` settings in the `InferenceService`. Creating additional head pods can cause them to be excluded from the Ray cluster.	
+* You have a Persistent Volume Claim (PVC) set up and configured for ReadWriteMany (RWX) access mode. 
+* You have set the storage protocol for the `StorageURI` to `PVC`.
+* Your Autoscaler is configured as `external`.
+
+== Creating the `vllm-multinode-runtime` ServingRuntime template
+
+Add the `vllm-multinode-runtime` ServingRuntime as a custom runtime to enable multi-node inferencing. The `vllm-multinode-runtime` ServingRuntime uses the same vLLM image as the `vllm-runtime` but it includes information necessary for multi-GPU inferencing.
+
+NOTE: Ensure that `TENSOR_PARALLEL_SIZE` and `PIPELINE_PARALLEL_SIZE` are not already configured through environment variables.
+
+.Procedure
+. In a terminal window, if you are not already logged in to your {openshift-platform} cluster as a cluster administrator, log in to the {openshift-platform} CLI as shown in the following example:
++
+[source]
+----
+$ oc login <openshift_cluster_url> -u <admin_username> -p <password>
+----
++
+. Select or create a namespace where you would like to deploy the model. In this example, `kserve-demo` is used and can be created using the following command:
++
+[source]
+----
+oc new-project kserve-demo
+----
++
```

Comment:
Create ServingRuntime using the multi node template:
```
oc process vllm-multinode-runtime-template -n redhat-ods-applications|oc apply  -f -
```

A `vllm-multinode-runtime` ServingRuntime will be created.